orbstation/code/objects/items/plushes.dm

```diff
@@ -17,3 +17,31 @@
 	attack_verb_simple = list("ohoho", "bite", "treat")
 	gender = FEMALE
 
+/obj/item/toy/plush/amoungplushie
+	name = "amoung plushie"
+	desc = "a cuddly toy depicting the dangerous and rare alien species the amoung, seems kinda sus."
+	icon = 'orbstation/icons/obj/items/plushes.dmi'
+	icon_state = "amoungplush"
+	attack_verb_continuous = list("suses", "stabs", "ejects")
+	attack_verb_simple = list("sus", "stab", "eject")
+	squeak_override = list('sound/creatures/alien_organ_cut.ogg' =1)
```

Comment:
```suggestion
	squeak_override = list('sound/creatures/alien_organ_cut.ogg' = 1)
```
and so on

Reply:
Done and done Jac, thanks for fixing the changelog btw